index.js

```diff
@@ -3,13 +3,19 @@
 const getStarredRepos = require('./lib/starred')
 const getTopic = require('./lib/topics')
 const { mapLimit } = require('async')
+const Spinnies = require('spinnies')
```

Comment:
Do you mind swapping `spinnies` in favour of https://github.com/sindresorhus/ora? `ora` seems to be more battle-tested and actively maintained.

Reply:
I tried to use ora initially but it is an ES Module that requires the import syntax.
What's your take on it?

Reply:
Ahh I see. How about https://www.npmjs.com/package/cli-spinner? I'm just worried that a less popular package does not work in all shells. Therefore I tend to a more wildly used version

Reply:
Converting the code base to es modulea sounds overkill given the lifetime value of this package

Reply:
> Ahh I see. How about https://www.npmjs.com/package/cli-spinner? I'm just worried that a less popular package does not work in all shells. Therefore I tend to a more wildly used version

Tested `cli-spinner` and it does not work as well due to the flow of your `console.log`. It is also unable to replace the loading indicator after it is complete. I would recommend sticking to `spinnies` as there isn't any issue with regards to compatibility.

<img width="742" alt="Screenshot 2021-10-03 at 6 11 16 AM" src="https://user-images.githubusercontent.com/26413686/135733052-664c847a-570a-41e6-ac5d-47089cb56f52.png">

Reply:
> It's 6ish am for me now, I will head to sleep soon... 😂

Ohh god, please do!